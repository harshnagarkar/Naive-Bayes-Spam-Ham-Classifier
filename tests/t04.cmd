  training_ham.txt testing_spam.txt testing_ham.txt training_spam.txt 1 
