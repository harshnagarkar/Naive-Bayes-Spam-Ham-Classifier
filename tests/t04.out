ham -182.544 -202.845
ham -124.001 -135.372
ham -275.724 -293.867
ham -491.326 -537.164
ham -352.149 -372.37
ham -269.397 -295.829
ham -265.254 -277.356
ham -197.624 -209.714
ham -352.362 -388.836
ham -225.78 -252.788
ham -351.786 -374.254
ham -362.798 -380.66
ham -229.649 -239.545
ham -480.179 -513.866
ham -0.164622 -1.88529
ham -408.664 -434.235
ham -1236.48 -1279.35
ham -4503.96 -4709.49
ham -1434.4 -1487.96
ham -2596.24 -2654.92
ham -485.297 -509.765
ham -295.225 -313.726
ham -931.14 -967.58
ham -455.677 -487.582
ham -1694.87 -1757.57
ham -589.506 -607.857
ham -790.278 -817.705
ham -5479.94 -5671.73
ham -4450.2 -4704.88
ham -2230.49 -2299.89
ham -1932.99 -2001.79
ham -276.592 -282.459
ham -2692.93 -2749.9
ham -0.164622 -1.88529
ham -0.164622 -1.88529
ham -0.164622 -1.88529
ham -0.164622 -1.88529
ham -0.164622 -1.88529
ham -0.164622 -1.88529
ham -0.164622 -1.88529
0 0.375 0.375
